app/manager-frontend/src/pages/ActivitySource/index.jsx

```diff
@@ -24,28 +24,51 @@ import { useScrollIntoView } from "helpers/hooks/useScrollIntoView";
 import { ResultsModifiers } from "components/resultsmodifiers/ResultsModifiers";
 
 export const ActivitySource = ({ activitySource, action, id }) => {
-  // TODO: Get this from the backend
   const typeOptions = [{ id: "RQUEST" }];
   const { data: datasourceOptions } = useDataSourceList();
   const { isOpen, onOpen, onClose } = useDisclosure();
   const { activitysource } = useBackendApi();
   const [feedback, setFeedback] = useState();
-  const submitText = !activitySource
-    ? "Create Activity Source"
-    : "Save changes";
-  const headingText = !activitySource
-    ? "Create a new Activity Source"
-    : "Edit Activity Source";
-  const isUpdate = !activitySource
-    ? false
-    : true
+  const [isLoading, setIsLoading] = useState();
+  const submitText = !activitySource ? "Create" : "Save";
+  const headingText = !activitySource ? (
+    <div>
+      <Text
+        color={"blue.500"}
+        fontWeight={600}
+        letterSpacing={1.1}
+        fontSize={"2xl"}
+        textTransform={"uppercase"}
+      >
+        Create Activity Source
+      </Text>
+    </div>
```

Comment:
```suggestion
    <Flex>
      <Text
        color={"blue.500"}
        fontWeight={600}
        letterSpacing={1.1}
        fontSize={"2xl"}
        textTransform={"uppercase"}
      >
        Create Activity Source
      </Text>
    </Flex>
```
prefer Chakra Components to html elements for consistency.

use `Box` for a block `div`; `Flex` for a flex `div`. 99% of the time I would use Flex. Or in this case, if it doesn't need to be wrapped (I appreciate it might), you could use `<Text as="div" ... />`